packages/paste-core/components/breadcrumb/src/index.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {Anchor} from '@twilio-paste/anchor';
+import {Text} from '@twilio-paste/text';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+
+const BreadcrumbSeparator: React.FC = () => (
+  <Text
+    as="span"
+    color="colorTextWeak"
+    fontSize="fontSize20"
+    lineHeight="lineHeight20"
+    paddingLeft="space20"
+    paddingRight="space20"
+    role="presentation"
+  >
+    /
+  </Text>
+);
+
+export interface BreadcrumbItemProps extends React.HTMLAttributes<HTMLLIElement> {
+  children: NonNullable<React.ReactNode>;
+  href?: string;
+  last?: boolean;
+}
+
+const BreadcrumbItem = React.forwardRef<HTMLAnchorElement, BreadcrumbItemProps>(
+  ({children, href, last, ...props}, ref) => {
+    return (
+      <Box
+        {...safelySpreadBoxProps(props)}
+        alignItems="center"
+        as="li"
+        color="colorText"
+        display="inline-flex"
+        fontSize="fontSize20"
+        lineHeight="lineHeight20"
+      >
+        {href ? (
+          <Anchor href={href} ref={ref}>
```

Comment:
Is it bad to forward a ref, but only render that component some of the time? If we're rendering Text, but still forwarding a ref, if I grab the ref from `BreadcrumbItem`...

Reply:
Basically, if I don't pass the `ref` to `Text` as well, all of the properties that comes with `ref` from `BreadcrumbItem` won't get passed.